werf.yaml

```diff
@@ -455,10 +455,14 @@ ansible:
         chdir: /dhctl
 ---
 image: dhctl-tests
+fromCacheVersion: "20220125"
 fromImage: base-for-go
 git:
 - add: /dhctl
   to: /dhctl
+  stageDependencies:
+    install:
+    - go.sum
```

Comment:
Please add Makefile as a stage dependency as well. According to the `make deps`, it downloads liner and other test requirements with curl, which is not dependent on the `go.sum` file.

Reply:
done